daweb/zaklady-gitu/uvod-do-gitu/dulezite-prikazy.md

```diff
@@ -4,13 +4,16 @@
 : Vytvoří lokální kopii vzdáleného repozitáře.
 
 **add**
-: Přidá změny do :term{cs="oblasti připravených změn" en="stage"}.
+: Přidá změny do :term{cs="oblasti připravených změn" en="stage, index"}.
```

Comment:
Název „index“ jsem pro stage nikdy neslyšel, to se používá?

Reply:
Používá se to primárně v dokumentaci (`man git-add`), v praxi se ten prostor nejpojmenovává vůbec a používají se jen slovesa a přídavný jména ("to stage", "staged changes" apod.) budu o tom trochu mluvit